doc/source/whatsnew/v1.1.0.rst

```diff
@@ -1132,6 +1132,7 @@ Groupby/resample/rolling
 - Bug in :meth:`core.groupby.DataFrameGroupBy.transform` when ``func='nunique'`` and columns are of type ``datetime64``, the result would also be of type ``datetime64`` instead of ``int64`` (:issue:`35109`)
 - Bug in :meth:`DataFrame.groupby` raising an ``AttributeError`` when selecting a column and aggregating with ``as_index=False`` (:issue:`35246`).
 - Bug in :meth:'DataFrameGroupBy.first' and :meth:'DataFrameGroupBy.last' that would raise an unnecessary ``ValueError`` when grouping on multiple ``Categoricals`` (:issue:`34951`)
+- Bug when combining methods :meth:`DataFrame.groupby` with :meth:`DataFrame.resample` and :meth:`DataFrame.interpolate` raising an ``TypeError`` (:issue:`35325`)
```

Comment:
I might do 
```
- Bug in `meth`:`Resampler.interpolate` when combining :meth:`DataFrame.groupby`, :meth:`DataFrame.resample` and :meth:`DataFrame.interpolate` was raising a ``TypeError`` (:issue:`35325`)
```

Reply:
> Release note might need to be moved to 1.2 (see #34730)

Thanks for the link. I was looking for the whatsnew 1.2 but have seen that the PR, which will add this, is still open.

Reply:
has now been merged